src/components/TableSearchBox.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import clearInputStyles from "./ClearInputButton.module.css";
+
+interface Props {
+  value: string;
+  onChange: (event:any) => void;
+  onClear: () => void;
+  placeholder: string
+}
+
+const TableSearchBox: React.FC<Props> = ({value, onChange, onClear, placeholder}) => {
+  return (
+    <div
+        style={{
+          // align search button on same line as input search field
+          display: "flex",
+          flexDirection: "row",
+          alignItems: "flex-start"
+        }}
+      >
+        <div
+          style={{
+            position: "relative",
+            display: "flex",
+            flexDirection: "row",
+            alignItems: "center"
+          }}
+        >
+          <input
+            type="text"
+            placeholder={placeholder}
+            value={value}
+            style={{
+              // make sure input field has same height as search button
+              padding: "6px",
+              paddingLeft: "35px",
+              paddingRight: "25px"
+            }}
+            onChange={onChange}
+          />
+
+          <i
+            id="searchBoxClearButton"
+            className={`${clearInputStyles.ClearInput} material-icons`}
+            style={
+              value === ""
+                ? { right: "6px", display: "none" }
+                : { right: "6px" }
+            }
+            onClick={onClear}
+          >
+            clear
+          </i>
```

Comment:
I think we have a component for the clear input button already. You can reuse it.